lib/task_listener.js

```diff
@@ -181,6 +182,18 @@ export default class TaskListener extends EventEmitter {
               stack: e.stack
           });
         }
+        let stats = {
+          uptime: this.host.billingCycleUptime(),
+          interval: this.host.billingCycleInterval()
+        };
+        let remainder = stats.interval - (stats.uptime % stats.interval);
+        if(remainder * this.runtime.taskQueue.slowdownDivisor > stats.interval) {
```

Comment:
we should add something to the task polling test to verify this logic and to make sure it doesn't accidentally get changed later on.